app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -0,0 +1,224 @@
+
+### Get help applying for teacher training
+
+To get advice about training to teach, you can:
+
+* call +44 (0) 800 389 2500, Monday to Friday between 8:30am and 5:30pm (UK local time) – calls are free within the UK but will otherwise be charged at your standard rate
+* use the live chat service at the bottom of this page
+* [sign up to get a dedicated adviser](https://adviser-getintoteaching.education.gov.uk/) – you’ll be asked for your phone number, email and location so a teacher training adviser can call you back in your time zone
+
+### Get help with international qualifications
+
+If your qualifications come from a non-UK institution, your teacher training provider may want to see a [statement of comparability](https://enic.org.uk/Qualifications/SOC/Default.aspx) from UK ENIC. A statement of comparability proves that your school and university qualifications are the same standard as UK GCSEs and a UK undergraduate degree.
+
+Call us on +44 800 389 2500 for:
+
+* guidance on the UK equivalents of your qualifications
+* a free statement of comparability, once you’ve submitted your application, if your provider asks for this
+* advice about what to do if you do not have English, maths and science qualifications of grade 4 GCSE standard
+
+
+### Find and apply for a teacher training course
+
+[Find postgraduate teacher training](https://www.gov.uk/find-postgraduate-teacher-training-courses) lists courses leading to qualified teacher status by region, subject and age group, and guides you to application.
+ 
+### If you do not have a university degree
+
+If you do not have a degree, you can [apply to study a 3 to 4 year undergraduate course](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes) in England which combines a teaching degree with [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts). When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
+
+The [UK Council for International Student Affairs](https://www.ukcisa.org.uk/) has lots of helpful information about studying at an English university.
+
+### Further information
+
+[Train to be a teacher](/train-to-be-a-teacher) gives more information about training in England. 
+
+[Funding your training](/funding-your-training) explains what financial support is available if you are in the UK under the [Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa), the [Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme) or the [Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme).
```

Comment:
Change to:
Find out what financial support is available to [fund your training](/funding-your-training) if you are in the UK under the [Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa), the [Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme) or the [Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme).